File: fce-extension/python/base/crypto.py
"""Cryptographic utilities shared across extensions."""

from Crypto.Hash import keccak


def keccak256(data: bytes) -> bytes:
    """Compute the Keccak-256 hash of data."""
    h = keccak.new(digest_bits=256)
    h.update(data)
    return h.digest()

File: fce-extension/fce-sign/typescript/src/base/crypto.ts
/** Cryptographic utilities shared across extensions. */

import { keccak_256 } from "@noble/hashes/sha3";

/** Compute the Keccak-256 hash of data. */
export function keccak256(data: Uint8Array): Uint8Array {
  return keccak_256(data);
}
